.circleci/config.yml

```diff
@@ -98,19 +120,32 @@ workflows:
       - build:
           requires:
             - linter
-          filters:
-            branches:
-              only: master
+
```

Comment:
By removing the filter we make sure that build step runs on PRs as well.
<img width="1274" alt="Screenshot 2024-11-20 at 2 36 16 PM" src="https://github.com/user-attachments/assets/b2b9f314-55f8-4fb9-920e-139d233523d8">